CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+### 1.7 (Next)
+
+**Bugs**
+
+* [#48](https://github.com/resourcelib/resourcelib/pull/56): Fixed `IconDirectoryResource.SaveTo` has bugs with some .ico file - [@jairbubbles](https://github.com/jairbubbles).
```

Comment:
The number is wrong, it says 48, but PR is 56 in the link. I know I know 🙄

Reply:
Eagle eye! I just indicated the issue number as it was a #. It was not consistent in the file so I mixed both. I'll put only issue number.